10.9-libcxx/stable/main/finkinfo/sci/pytables-py.info

```diff
@@ -21,13 +41,18 @@ CompileScript: <<
  %p/bin/python%type_raw[python] setup.py build --hdf5=%p/ --lzo=%p/ --bzip2=%p/ 
  cd doc
  if [ %type_pkg[python] -lt 30 ]; then
-  export PYTHONPATH=$(ls -d %b/build/lib.macosx-*-%type_raw[python])
-  make SPHINXBUILD=sphinx-build2.7 html
+  mkdir -p build/html
+  cat <<EOF > build/html/README
+  The PyTables HTML documentation currently cannot be built under Sphinx v1.8.4 due to
+  Sphinx errors.
+EOF
+#  export PYTHONPATH=$(ls -d %b/build/lib.macosx-*-%type_raw[python])
+#  make SPHINXBUILD=sphinx-build2.7 html
  else
   mkdir -p build/html
   cat <<EOF > build/html/README
   The PyTables HTML documentation currently cannot be built under Python3 due to
-  Sphinx errors. Please install the %N-py27 package for the offline docs.
```

Comment:
The original motivation for this was that sphinx-py27 was working while the other variants were not.
With the docs build disabled entirely this does not make sense:
```
> cat /sw/share/doc/pytables-py36/html/README                                                                                          
  The PyTables HTML documentation currently cannot be built under Python3 due to
  Sphinx errors. Please install the pytables-py36 package for the offline docs.
```
so the last sentence should be removed or changed to `Please refer to https://www.pytables.org for the online docs.` In fact the entire test for `%type_pkg[python] -lt 30` can be removed as currently all variants are failing the same - the Sphinx `Extension error:
Could not import extension IPython.sphinxext.ipython_console_highlighting (exception: cannot import name 'path')` being due to a too old or missing ipython.
So in fact the whole doc build can be fixed by adding `ipython-py%type_pkg[python] (>= 5.7.0-1), sphinx-py%type_pkg[python] (>= 1.8.4-1)` to `BuildDepends`, but for that it seems I have to fix a number of conflicts in #164.

Reply:
I was simply too hesitating to remove stuff from a package description of someone else. As you think, this is fine, i did so in commit c579b0d.